src/logic/contractInteraction/sources/ABIService/index.ts

```diff
@@ -35,6 +35,10 @@ export const isAllowedMethod = ({ name, type }: AbiItem): boolean => {
 }
 
 export const getMethodAction = ({ stateMutability }: AbiItem): 'read' | 'write' => {
+  if (!stateMutability) {
+    return 'write'
+  }
```

Comment:
Shouldn't ```includes``` method already check this? If it is ```null``` or ```undefined``` should return false, so in shorthand return that is ```write```.

Anything I'm not seeing here?

Reply:
Undefined is not a valid param for .includes

Reply:
Not on strict typescript, ok